server/server.go

```diff
@@ -159,7 +159,7 @@ func init() {
 	maxConcurrentLoginRequestsCount = env.ParseNumFromEnv(maxConcurrentLoginRequestsCountEnv, maxConcurrentLoginRequestsCount, 0, math.MaxInt32)
 	replicasCount = env.ParseNumFromEnv(replicasCountEnv, replicasCount, 0, math.MaxInt32)
 	if replicasCount > 0 {
-		maxConcurrentLoginRequestsCount = maxConcurrentLoginRequestsCount / replicasCount
+		maxConcurrentLoginRequestsCount /= replicasCount
```

Comment:
Could we disable `assignOp` and add a comment that it's intentionally disabled for readability? It might just be me, but I find this to be quite a bit less readable.